### PR TITLE
Document Review - Publish Hybrid to App Store

### PR DESCRIPTION
**Pull request by MarkvanMents**
Documentation Team internal review

### CODE REVIEW

File: content/howto/mobile/publishing-a-mendix-hybrid-mobile-app-in-mobile-app-stores.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: "Publish a Mendix Hybrid Mobile App in Mobile App Stores"
 parent: "hybrid-mobile"
 menu_order: 60
-tags: ["mobile", "app store", "phonegap"]
+tags: ["mobile", "app store", "phonegap", "Hybrid"]
```
Comment (MarkvanMents) [inline content/howto/mobile/publishing-a-mendix-hybrid-mobile-app-in-mobile-app-stores.md]:
```suggestion
tags: ["mobile", "app store", "phonegap", "Hybrid"]
```

**Comment by MarkvanMents** [comment]
Have done the review in Word as cannot simply add just comments to the document.

Leaving open for potential text review.

**Comment by MarkvanMents** [comment]
@ConnorLand 
Do you want me to keep this Pull Request for your changes, or will you make a new one when you are ready?

**Comment by ConnorLand** [comment]
> @ConnorLand
> Do you want me to keep this Pull Request for your changes, or will you make a new one when you are ready?

I'd like to keep this PR around. I'll use it to implement the needed changes. Thank you.

**Comment by MarkvanMents** [comment]
@ConnorLand 

Can I kill this branch and Pull Request now?

**Comment by ConnorLand** [comment]
> @ConnorLand
> 
> Can I kill this branch and Pull Request now?

Yes please.

**Comment by MarkvanMents** [comment]
Work done on another branch. This PR no longer needed.

**Pull request closed without merging**